low-level/files/opt/seagate/sspl/bin/manifest_support_bundle

```diff
@@ -23,213 +23,230 @@ import sys
 import socket
 import json
 import subprocess
+from sspl_constants import PRODUCT_FAMILY, node_key_id
 
-from sspl_constants import PRODUCT_FAMILY
-
-localTempPath = "/tmp/support_bundle/"
```

Comment:
this whole code restructuring is making big change set visible, very hard to track what all changed. I believe putting these vars & functions inside class & then adding proper spaces are only involved.

Reply:
Yes @cdeshmukh